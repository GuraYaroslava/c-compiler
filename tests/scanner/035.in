/* comment */

a

/*
comment
*/

/* comment


*/

b

// comment

c

// c // o // m // m // e // n // t

d

// */          // comment, not syntax error

/*//*/ l();    // equivalent to l();

f = g/**//h;   // equivalent to f = g / h;

//\