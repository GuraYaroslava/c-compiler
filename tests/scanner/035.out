type: IDENTIFIER		ln: 3		pos: 1		lx: a		val: a

type: IDENTIFIER		ln: 14		pos: 1		lx: b		val: b

type: IDENTIFIER		ln: 18		pos: 1		lx: c		val: c

type: IDENTIFIER		ln: 22		pos: 1		lx: d		val: d

type: IDENTIFIER		ln: 26		pos: 8		lx: l		val: l

type: OPERATOR		ln: 26		pos: 9		lx: (		val: (

type: OPERATOR		ln: 26		pos: 10		lx: )		val: )

type: OPERATOR		ln: 26		pos: 11		lx: ;		val: ;

type: IDENTIFIER		ln: 28		pos: 1		lx: f		val: f

type: OPERATOR		ln: 28		pos: 3		lx: =		val: =

type: IDENTIFIER		ln: 28		pos: 5		lx: g		val: g

type: OPERATOR		ln: 28		pos: 10		lx: /		val: /

type: IDENTIFIER		ln: 28		pos: 11		lx: h		val: h

type: OPERATOR		ln: 28		pos: 12		lx: ;		val: ;

